packages/starknet-snap/src/utils/formatter-utils.test.ts

```diff
@@ -65,3 +72,92 @@ describe('mapDeprecatedParams', () => {
     expect(requestParams).toStrictEqual(expected);
   });
 });
+
+describe('callToTransactionReqCall', () => {
```

Comment:
nit:
the test should be more realistic and we can re-use some existing constant